### DISCUSSION TITLE
Drive level with/without PA

### DISCUSSION THREAD

**Discussion by HB9BMR** [ExpertSDR3/ExpertSDR3-BUG-TRACKER#437] [Ideas]
Drive level is stored per MODE and per band.
I suggest to add the possibility that when PA is on, I can store lower levels without changing the the above stored levels when PA off.
My PA only needs 10 percent drive level.

Background:
I normally work with 100 W no PA
When conditions demand, I like to push PA and the driive level should go automatically to the lower level that I have stored for working with PA.

73, Lorenz, HB9BMR

**Comment by DK8JV** [comment]
I support this suggestion. It's something that I have been wishing for for a long time.

Best 73
Eberhard, DK8JV
